testing/src/main/scala/sbt/JUnitXmlTestsListener.scala

```diff
@@ -211,3 +216,11 @@ class JUnitXmlTestsListener(val outputDir: String) extends TestsListener {
   /**Returns None*/
   override def contentLogger(test: TestDefinition): Option[ContentLogger] = None
 }
+
+object JUnitXmlTestsListener {
+  private val dateFormat = new SimpleDateFormat("yyyy-MM-dd")
+  private val timeFormat = new SimpleDateFormat("HH:mm:ss")
+
+  def formattedTimestamp(timestamp: Long): String =
+    s"${dateFormat.format(timestamp)}T${timeFormat.format(timestamp)}"
```

Comment:
Here's more JDK 8-ish time formatting code.


```scala
scala> import java.time.LocalDateTime
import java.time.LocalDateTime

scala> import java.time.format.DateTimeFormatter
import java.time.format.DateTimeFormatter

scala> import java.time.temporal.ChronoUnit
import java.time.temporal.ChronoUnit

scala> val t = LocalDateTime.now().truncatedTo(ChronoUnit.SECONDS)
t: java.time.LocalDateTime = 2018-05-12T01:39:18

scala> t.format(DateTimeFormatter.ISO_LOCAL_DATE_TIME)
res0: String = 2018-05-12T01:39:18
```

Reply:
Hmm, not sure if it's absolutely necessary to truncate to seconds. Wdyt?

Reply:
Oh, yep, it is. :) Good catch.

https://github.com/windyroad/JUnit-Schema/blob/master/JUnit.xsd#L15